src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSGapicSurfaceTestTransformer.java

```diff
@@ -126,10 +127,17 @@ private MockCombinedView generateTestView(
               .build());
     }
 
+    String outputPath = "test";
```

Comment:
Let's move this to a separate method.

If you look at https://github.com/googleapis/googleapis/blob/master/gapic/api/artman_videointelligence.yaml#L56,

The test file name should be gapic-<version>.js